services/sensors/sensors.go

```diff
@@ -21,15 +21,17 @@ func init() {
 }
 
 // A Readings ties both the sensor name and its reading together.
-//
-//	// Get the readings provided by the sensor.
-//	readings, err := mySensor.Readings(context.Background(), nil)
 type Readings struct {
 	Name     resource.Name
 	Readings map[string]interface{}
 }
 
 // A Service centralizes all sensors into one place.
+//
```

Comment:
This is looking great! I'll give this a full read-through Monday, but caught this: This code sample for `Readings` should precede the description to `Service` + a newline. Currently it will appear as part of the `Service` method description.